maven-bom/pom.xml

```diff
@@ -204,6 +204,14 @@ under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
```

Comment:
This is a no go

Reply:
@cstamas removed. hope you find another better way for resolving this.